libs/blocks/src/lib/fast-payment/base/index.tsx

```diff
@@ -14,43 +20,45 @@ const Base: React.FC<FastPaymentProps> = ({
   className,
 }) => {
   return (
-    <FluidContainer
-      className={qtJoin(
-        'flex flex-col items-center py-general-4xl text-center',
-        className,
-      )}
-      data-testid="fast-payment-container"
-    >
-      <Heading.H2>{title}</Heading.H2>
-      {description && (
-        <Text data-testid="fast-payment-description" className="pt-general-md">
-          {description}
-        </Text>
-      )}
-      {link && (
-        <CustomLink
-          href={link.href}
-          className="w-fit pt-general-xl"
-          hasIcon
-          data-testid="fast-payment-link"
-        >
-          {link.content}
-        </CustomLink>
-      )}
-      <CardsContainer
-        cards={cards}
-        variant="ContentBottom"
-        cols={cols}
-        dense={dense}
-        className="pt-general-2xl"
-        sliderClass={sliderClass}
-      />
-      {disclaimer && (
-        <Text size="sm" className="pt-general-2xl text-typography-subtle">
-          {disclaimer}
-        </Text>
-      )}
-    </FluidContainer>
+    <Section className="overflow-hidden py-general-4xl text-center">
+      <FluidContainer
+        className={qtJoin('flex flex-col items-center', className)}
```

Comment:
move the qtJoin to section and use qtMerge here

Reply:
done